### ISSUE TITLE
Capitellum fracture

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10794]
**Preferred term label:**
Capitellum fracture

**Synonyms**
capitellar fracture

**Definition (free text, please give PubMed ID)**
fracture of the capitellum

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Capitellum fracture
Radial head fracture

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001140
- label: Capitellum fracture
- parents: [Fractured radius (HP:0003978)](https://hpo.jax.org/app/browse/term/HP:0003978)
- pmid: 
- def: A partial or complete breakage of the capitulum of the humerus, which is a smooth, rounded eminence on the lateral portion of the distal articular surface of the humerus.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10794
- synonyms: Capitellar fracture(NONE)

**Comment by grahamponting** [comment]
Hi, on reviewing the recent release, I find this concept (Capitellum fracture, HP:6001140) has been attached to the parent of "Fractured radius HP:0003978", but the capitellum is part of the humerus, as stated in the definition above. Could you review please.
Graham Ponting

**Comment by pnrobinson** [comment]
reopening to check.

**Comment by pnrobinson** [comment]
Thanks, I have moved to be a child of fracture humerus, this will be visible in the next release!
